File: PRAK304-2210817220004-Melisa Anggini.c
#include<stdio.h>
int main()
{
    int a;
    scanf("%i",&a);
    if(a>=1 && a<=9){
        printf("Satuan\n");
    }
    else if(a>10 && a<=19){
        printf("Belasan\n");
    }
    else if(a>=20 && a<=99){
        printf("Puluhan\n");
    }
    else if(a<1){
        printf("Nol\n");
    }
    else{
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
}

File: PRAK304-2210817220004-Melisa Anggini.py
a=int(input())
if a>=1 and a<=10:
    print("Satuan")
elif(a>=10 and a<=19):
    print("Belasan")
elif(a>=20 and a<=99):
    print("Puluhan")
elif a<1:
    print("Nol")
else:
    print("Anda Menginput Melebihi Limit Bilangan")